hybrisCustom/custom-ext/bonstore/bonstorecore/src/org/bonstore/core/dao/impl/OrganizationDaoImpl.java

```diff
@@ -0,0 +1,44 @@
+/**
+ *
+ */
+package org.bonstore.core.dao.impl;
+
+import de.hybris.platform.servicelayer.search.FlexibleSearchQuery;
+import de.hybris.platform.servicelayer.search.FlexibleSearchService;
+
+import java.util.List;
+
+import org.bonstore.core.dao.OrganizationDao;
+import org.bonstore.core.model.OrganizationModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+
+/**
+ * @author Tanmoy_Mondal
+ *
+ */
+@Component(value = "organizationDao")
```

Comment:
It's recommendet to register bean in xml file. Please refer the same link on kb.epam.com